src/main/java/Duke.java

```diff
@@ -1,10 +1,179 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.File;
+
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Your Only Friend\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n");
+
+        ArrayList<Task> listOfTasks = new ArrayList<>();
+
+        load(listOfTasks);
+
+        Scanner whatToDo = new Scanner(System.in);
+        String whatToDoCall = whatToDo.nextLine().trim().toLowerCase();
```

Comment:
Rather strange names imo, maybe can consider using inputScanner and input respectively